Core/Core/Logging/Setup.cs

```diff
@@ -40,7 +40,11 @@ static Setup()
   /// </summary>
   internal static string VersionedHostApplication { get; private set; } = HostApplications.Other.Slug;
 
-  public static void Init(string versionedHostApplication, string hostApplication)
+  public static void Init(
+    string versionedHostApplication,
+    string hostApplication,
+    SpeckleLogConfiguration logConfiguration = null
```

Comment:
@JR-Morgan file did not have nullability activated, so I decided to leave this input "non-nullable" but defaulting to null....

`SpeckleLog.Init` does take a nullable `SpeckleLogConfiguration?`.

Reply:
That's fine. In a non-nullable context, this is a Null oblivious type.